test.sh

```diff
@@ -74,6 +74,10 @@ if [[ $dnssec_validated != 0 ]]; then echo PASS; else retval=1; echo FAIL; fi
 
 
 # Test 2
+echo -n "Retrieve EFS ID for cluster... "
```

Comment:
Why are we setting up the driver, PV, and storageClass in the test? Seems like that's something the administrator would do when provisioning the EKS cluster, since the administrator knows then what the EFS ID is, and the person just using the brokered EKS later doesn't know anything about it. 

In particular, the "admin" is us driving this process via Terraform so we can use these resources:
https://registry.terraform.io/providers/hashicorp/kubernetes/latest/docs/resources/csi_driver_v1
https://registry.terraform.io/providers/hashicorp/kubernetes/latest/docs/resources/storage_class
https://registry.terraform.io/providers/hashicorp/kubernetes/latest/docs/resources/persistent_volume

Reply:
Elaborating: The test should only cover creating the claim and pod, since that's all that users of a brokered k8s are expected to do. Similarly the solr-operator handles creating a claim and associating that claim with a pod while it's provisioning a SolrCloud instance.

Reply:
(Hence there's no need to be mucking with the EFS ID here.)